brewdo

```diff
@@ -240,7 +240,7 @@ def cmd_brew(*args)
   end
 
   # some Homebrew commands can be called directly
```

Comment:
Let's amend this comment to something like
` # some Homebrew commands can be called directly, as long as we're not running as root`